### DISCUSSION TITLE
map refresh while navigation is delayed

### DISCUSSION THREAD

**Discussion by Jogypilote** [osmandapp/OsmAnd#18706] [General]
Hi all,
I use a  new Samsung galaxy A54 for navigation at my car and motorbike.
Everything works quite well  but the map moves while navigation with a delay of 50 to 100 m and than it jumps to the new position The rotation of the map happens immediately.  In a bad case you miss the right corner to turn.
The software is Android 13 and Osmand + 4.5.10
Is there anything in the configuration to adjust?
Any help is appreciated .
Greetings Jörg

**Comment by mariush444** [comment]
Try to play with options in profile conf: Kalman and animate own position.

**Comment by Jogypilote** [comment]
thank you for the advice. 
I will test it in the next days. 
Joerg

Am 10. Dezember 2023 16:24:58 MEZ schrieb Mars Yush ***@***.***>:

**Comment by ThomasMeister123** [comment]
Same here. Interesting: When recording a track, the trackline is ALWAYS in the right position while the "car" in the display is 100m behind und then "jumps" to the right position

**Comment by vshcherb** [comment]
Animate own position should have 1 second delay but provide non-jump animation. Wihtout it changes are instantaneous but no-smooth animation

**Reply by Jogypilote** [reply]
This I will try.
Before I tried Kalman-filter off and on. On is better and No animation on is better.
But especially at a road curve with low speed (<50 kmh) it becomes  delayed.

**Comment by ThomasMeister123** [comment]
Here an example of this behavior.

Real position ist at (1). OSMand already knows this (see green track-recording-line). Also the rotation of the map is for the real position.

And thats not a "worse" example. Sometimes the "gap" is much longer

![2024-11-12 12_08_23-XnView -  A2 png ___ 720x1280 ___ Verschluss_  ___ Blende_  ___ Iso_ ](https://github.com/user-attachments/assets/475dfd2b-a8a3-48f4-a012-fb6035f68f49)
